skills/lark-base/references/lark-base-form-detail.md

```
@@ -0,0 +1,318 @@
+|------|------|------|
+| `--share-token <token>` | 是 | 表单分享 Token（从表单分享链接中提取） |
+| `--format` | 否 | 输出格式：json（默认）\| pretty \| table \| ndjson \| csv |
+| `--as` | 否 | 身份：user（默认）\| bot |
+| `--dry-run` | 否 | 预览 API 调用，不执行 |
+| `--jq <expr>` | 否 | 用 jq 表达式过滤 JSON 输出 |
+
+### 从分享链接提取 share-token
+
+用户提供形如以下格式的表单分享链接时：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)